text_formatter.go

```diff
@@ -138,6 +166,13 @@ func (f *TextFormatter) printColored(b *bytes.Buffer, entry *Entry, keys []strin
 	}
 }
 
+func (f *TextFormatter) getLevelColor(l Level) uint8 {
+	if val, ok := f.colorMap[l]; ok {
+		return val
+	}
+	return gray
```

Comment:
This remains unconfigurable. How about having a `DefaultColor` in `colorMap`?

Reply:
"DefaultColor" as part of the color map is because of the level to color mapping (type colorMap map[Level]uint8) currently not good in my opinion. I would let the default color setting like it is for now and only implement a "DefaultColor handling" later if it is really necessary.